starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +18,43 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+function findLongestWord(arr1){
+  var longestWord = '';
+  if (longestWord = ''){
+    longestWord = undefined;
+  }
+  else{
+  for (var i =0; i < arr1.length; i++){
+    if (longestWord.length < arr1[i].length){
+      longestWord = arr1[i];
+    }
+  }}
+  return longestWord;
+}
 
+// Calculating a Sum
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray([]){
+  var sumCount = 0;
+  [].forEach(function(number){
+  sumCount += number;
+    });
+  return sumCount;
+}
+
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers([]){
```

Comment:
Hmm? I wonder why you think you would need to keep passing empty arrays as arguments for functions? This is incorrect. If you need help clearing this up, Slack me and we can sit down and work through it.

Here is the correct method, so you can compare:
```js
// Calculate the Average
function averageNumbers(array) {
  if (array.length === 0) {
    return undefined;
  }
  console.log(sumArray(array), array.length);
  return sumArray(array) / array.length;
}

var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
var average = averageNumbers(numbersAvg);
console.log(average);
```